index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body style="background-image: url('assets\headerBG.png');">
+    <header>
+        <a href="index.html"><h1 class="logoText">TS</h1></a>
+        <nav class="headerNav"> 
+            <ul>
+                <li> <a  href="portfolio.html">Portfolio</a></li>  
+                <li><a href="experience.html">Experience</a></li>
+                <li> <a href="bio.html">Bio</a></li>
+                <li><a  class = "active" href="index.html">Home</a></li>
+            </ul>
+        </nav>
+    </header>
+    
+    <main>
+        <h2>Taj Smith</h2>
+        <hr>
+
+        <h4 id = "quoteText" >"Insert inspirational quote here"</h4>
+        <h5 id = "quoteSubText">-By Quote Maker</h5>
+
+        <img id = "homePic" alt="sunset" src="assets\sunset.jpg" height = 400px width = 400px>
+        <p>Welcome to my online portfolio!<br>Lorem ipsum dolor sit amet, tempor prodesset eos no. Temporibus necessitatibus sea ei, at tantas oporteat nam. Lorem ipsum dolor sit amet, tempor prodesset eos no.</p>
+    
```

Comment:
nit: consider removing extra blank lines if they aren't separating distinct sections of code